### PR TITLE
Validated the sag-is-stateful-cluster

### PR DESCRIPTION
**Pull request by PrajwalNagabhushana**
Added  integrationServer-${instance.name} as the clustering configuration was not setup.

### CODE REVIEW

File: templates/sag-is-stateful-cluster/README.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* Command Central 10.2 and higher
+
+### Supported platforms
+
+All supported Windows and UNIX platforms.
+All supported Database servers.
+
+### Supported use cases
+
+* Provisioning of new 9.8 or higher single server environment with database
```
Comment (YanaSimeonova) [inline templates/sag-is-stateful-cluster/README.md]: Remove it since it is a cluster env

File: templates/sag-is-stateful-cluster/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+## Creating a new stack with Integration Cluster layer using Web UI
+
+Consult [Creating a stack using Command Central Web UI](https://github.com/SoftwareAG/sagdevops-templates/wiki/Using-default-templates#creating-a-new-stack-using-web-ui)
+for additional information about using Stacks UI.
+
+* Open Stacks UI
+* Add new stack by clicking `(+)` icon
+* Add new Infrastructure layer
+* Add new Runtime layer
+  * Select `IS-SERVER` layer definition
```
Comment (sergeipogrebnyak) [inline templates/sag-is-stateful-cluster/README.md]: That's not a correct layer id for this type

File: templates/sag-is-stateful-cluster/template.yaml [hunk 3]
```diff
@@ -90,14 +90,15 @@ templates:
                 COMMON-MEMORY:
                   InitSize: ${is.memory.init}
                   MaxSize:  ${is.memory.max}
+            integrationServer-${instance.name}:                  
               COMMON-CLUSTER:
                 COMMON-CLUSTER:
-                  Name: ${is.cluster.name}
+                  Name: ${instance.name}-cluster
```
[the remark this answers is not in the document]
Comment (sergeipogrebnyak) [reply templates/sag-is-stateful-cluster/template.yaml]: What is the importance of the cluster name?

File: templates/sag-is-stateful-cluster/README.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+## Provisioning of new Cluster
+
+Consult [Applying template using Command Central CLI](https://github.com/SoftwareAG/sagdevops-templates/wiki/Using-default-templates#applying-template-using-command-central-cli) for additional information about applying templates.
+
+Provision `default` instance of Integration Server 10.1 with all latest fixes onto `dev1` and `dev2` nodes,
+connected to an Oracle database, listening on default ports 5555, 9999 and 8094 (jmx), with 512mb of memory:
+
+```bash
+sagcc exec templates composite apply sag-is-server nodes=dev1,dev2 \
```
[the remark this answers is not in the document]
Comment (PrajwalNagabhushana) [reply templates/sag-is-stateful-cluster/README.md]: I have updated.

File: templates/sag-is-stateful-cluster/README.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+## Local development and testing using Docker
+
+### With SQL Server database
+
+Provision [SQL Server](../sag-db-sqlserver) container and schemas
+
+Provision My webMethods Server:
+
+```bash
+CC_ENV=sqlserver ./provisionw sag-is-cluster
```
[the remark this answers is not in the document]
Comment (PrajwalNagabhushana) [reply templates/sag-is-stateful-cluster/README.md]: Updated.

File: templates/sag-is-stateful-cluster/README.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+### Supported use cases
+
+* Provisioning of new 9.8 or higher single server environment with database
+* Provisioning of new 9.8 or higher cluster environment
+* Installing latest fixes and support patches
+* Configuration of:
+  * License
+  * JVM memory
+  * Primary, diagnostics and JMX ports
+
```
[the remark this answers is not in the document]
Comment (PrajwalNagabhushana) [reply templates/sag-is-stateful-cluster/README.md]: Added

File: templates/sag-is-stateful-cluster/README.md (hunk 2 above)
Comment (PrajwalNagabhushana) [reply templates/sag-is-stateful-cluster/README.md]: Corrected.

**Comment by PrajwalNagabhushana** [comment]
IS there any thing I have to update ?
Please let me know.

**Comment by YanaSimeonova** [comment]
The layer definition is again not correct. Name it sag-is-stateful-cluster and i will take care of sag-cc-layer-defs template.

**Comment by PrajwalNagabhushana** [comment]
Renamed it to "sag-is-stateful-cluster". please take care of the yaml file as well.

**Pull request merged**